### DISCUSSION TITLE
Should I source /etc/zshrc

### DISCUSSION THREAD

**Discussion by zno5** [ohmyzsh/ohmyzsh#10208] [Help] [answered]
I'm using macOS big sur 11.5.2 and zsh 5.8

I wrote ". /etc/bashrc" first line in ~/.bashrc when I used bash.
I have a question.
Should I write that in my .zshrc which is generated by oh-my-zsh?
like this ". /etc/zshrc"

**Comment by mcornella** [comment, marked as answer]
`/etc/zshrc` should be automatically read by zsh when starting. You can verify that by running `zsh -xvic exit 2>&1 | less` which dumps the lines Zsh is running. Another option is to modify it to `echo` something specific and it should appear in the terminal when you open it.

Normally it is read before your home directory `.zshrc` file.

**Reply by zno5** [reply]
Thank you for your answer.
I followed your command and found that /etc/zshrc is first loaded.

I have another question.
Where should I write it to load /etc/zshrc if it were not loaded?
.zprofile or .zshrc?

**Reply by mcornella** [reply]
`.zshrc`, preferably at the very first line.

**Reply by zno5** [reply]
Thank you so much for your help.
